eZ/Publish/Core/FieldType/ImageAsset/Value.php

```diff
@@ -19,13 +19,22 @@ class Value extends BaseValue
      */
     public $destinationContentId;
 
+    /**
+     * The alternative image text (for example "Picture of an apple.").
+     *
+     * @var string|null
+     */
+    public $alternativeText;
```

Comment:
Continuing my comment about validating internal type of this member, well... I'm quite unsure, because my suggestion from before introduces a bit of inconsistency with regards to other member or other Field Types.
So not a request for change, but for a bit of discussion ;)

Reply:
Yet another comment - so maybe if alternativeText is not empty it should be returned as a result of `__toString()`? I think we still are allowed to change it before 2.3 release. 
Unless of course result of `__toString()` is not used somewhere as ID.

Reply:
This should be consistent with `ezrelation`  so current form is correct (see `\eZ\Publish\Core\FieldType\Relation\Value::__toString`).